packages/synapse-react-client/src/components/CardContainer/CardContainer.tsx

```diff
@@ -90,72 +96,168 @@ function _CardContainer(props: CardContainerProps) {
     ids,
     type: 'ENTITY_HEADER',
   })
+
+  const observationColIndex = rowSet?.headers?.findIndex(
```

Comment:
I don't think this logic belongs here.  Am I correct in assuming that in this case it will render many ObservationCard components?  Could this logic be moved to the ObservationCard (if it should be filtered out, then may it returns an empty react fragment?)?

Reply:
At the very least, this logic should be less specific to ObservationCards, and should utilize the Synapse tables faceted column features to get filter values and apply the filter to the rows server-side.

Reply:
I think a good example of a similar component is the `QuerySortSelector`